src/Symfony/Component/Messenger/Command/FailedMessagesRetryCommand.php

```diff
@@ -165,6 +167,16 @@ private function runInteractive(string $failureTransportName, SymfonyStyle $io,
 
     private function runWorker(string $failureTransportName, ReceiverInterface $receiver, SymfonyStyle $io, bool $shouldForce): int
     {
```

Comment:
@lyrixx Unfortunately no, the signals from the `SignalableInterface` are added before the `WorkerStartedEvent` is triggered. Since the `StopWorkerOnSignalsListener` overrides the signals, it wouldn't work.

Reply:
I mean, we could rely entirely on the console component, and not implement things here.

Reply:
So if I understood you correctly, you would replace the `StopWorkerOnSignalsListener` with the `SignalableCommandInterface`? This would then also need to be implemented on the `ConsumeMessagesCommand` as well.

Reply:
yes 👍🏼

Reply:
@lyrixx Done, the idea is that if the console component is installed, the listener doesn't get registered.